js/ui/marker.js

```diff
@@ -129,40 +132,16 @@ Marker.prototype = {
      * @returns {Marker} `this`
      */
     togglePopup: function () {
-        if (this._popup) {
-            if (this._popup._map) {
-                this._closePopup();
-            } else {
-                this._openPopup();
```

Comment:
> This will throw an error because `_openPopup` is expecting an event argument.

https://github.com/mapbox/mapbox-gl-js/pull/3056#discussion_r76332587